webapp/pages/admin/users.vue

```diff
@@ -1,24 +1,23 @@
 <template>
-  <div>
-    <ds-space>
-      <ds-card :header="$t('admin.users.name')">
-        <ds-form v-model="form" @submit="submit">
-          <ds-flex gutter="small">
-            <ds-flex-item width="90%">
-              <ds-input
-                model="query"
-                :placeholder="$t('admin.users.form.placeholder')"
-                icon="search"
-              />
-            </ds-flex-item>
-            <ds-flex-item width="30px">
-              <base-button filled circle type="submit" icon="search" :loading="$apollo.loading" />
-            </ds-flex-item>
-          </ds-flex>
-        </ds-form>
-      </ds-card>
-    </ds-space>
-    <ds-card v-if="User && User.length">
+  <div class="admin-users">
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-12T13:37:38Z" title="Wednesday, February 12th 2020, 2:37:38 pm +01:00">Feb 12, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,24 +1,23 @@
 <template>
-  <div>
-    <ds-space>
-      <ds-card :header="$t('admin.users.name')">
-        <ds-form v-model="form" @submit="submit">
-          <ds-flex gutter="small">
-            <ds-flex-item width="90%">
-              <ds-input
-                model="query"
-                :placeholder="$t('admin.users.form.placeholder')"
-                icon="search"
-              />
-            </ds-flex-item>
-            <ds-flex-item width="30px">
-              <base-button filled circle type="submit" icon="search" :loading="$apollo.loading" />
-            </ds-flex-item>
-          </ds-flex>
-        </ds-form>
-      </ds-card>
-    </ds-space>
-    <ds-card v-if="User && User.length">
+  <div class="admin-users">
```


could this be `article`?

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-17T09:43:31Z" title="Monday, February 17th 2020, 10:43:31 am +01:00">Feb 17, 2020</time>_

---


I'd probably go for `section` rather than `article` but either would work. However, I'd prefer to not have this element there at all! That would have required bigger changes, though and at some point I just had to stop myself because this PR was already getting way too big... I'll keep it for next time, ok? :grin: